pyproject.toml

```
@@ -17,7 +17,7 @@ kasa = "kasa.cli:cli"
 [tool.poetry.dependencies]
 python = "^3.7"
 importlib-metadata = "*"
-asyncclick = "^7"
+asyncclick = "^8"
```

```suggestion
asyncclick = ">=7"
```
let's make it the minimum version so that there's no need to bump it regularly. Also, please run `poetry update` and add the lock file to the PR to fix the CI.